src/boot/efi/util.c

```diff
@@ -265,26 +265,36 @@ char16_t *xstr8_to_path(const char *str8) {
         return path;
 }
 
-void mangle_stub_cmdline(char16_t *cmdline) {
-        char16_t *p = cmdline;
+static bool shall_be_whitespace(char16_t c) {
+        return c <= 0x20U; /* All control characters + space */
```

Comment:
Didn't you once note that it should mangle `0x7F` too?

Reply:
true